sources/logdog/conf/logdog.aws-k8s-1.19.conf

```diff
@@ -0,0 +1 @@
+aws-k8s.conf
```

Comment:
Hopefully the error thrown when this was missing was not too hard to decipher?

Reply:
I didn't actually encounter any errors. This is here because it was in the 1.18 PR. I merely mimicked it :)